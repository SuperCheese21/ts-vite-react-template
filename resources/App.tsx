export const App = (): JSX.Element => (
  <>
    <button className="btn btn-primary">Button</button>
  </>
);

export default App;
