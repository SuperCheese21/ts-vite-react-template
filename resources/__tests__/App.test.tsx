import { render } from '@testing-library/react';
import { expect, it } from 'vitest';
import App from '../App';

it('renders app', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
