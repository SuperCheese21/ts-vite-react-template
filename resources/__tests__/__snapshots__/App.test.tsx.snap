// Vitest Snapshot v1

exports[`renders app 1`] = `
<div>
  <button
    class="btn btn-primary"
  >
    Button
  </button>
</div>
`;
